backend/src/main/cards/card.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC
+
+
+class Card(ABC):  # pylint:disable=too-few-public-methods
```

Comment:
Remove `ABC` as this class is not an abstract class nor a base class
I imagine this is from when Discard, Hand, and Lost each derived from Card but is not necessary anymore